entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I went with this way because github actions supports 2 types of distributable actions: node based and docker based. Since I'm using a python script to do the testing, that means I need to distribute it as a docker based action.

I haven't found a way to get the 2 docker images (the deephaven one and the testing one) to communicate properly, except for bundling them under the same project name by using `docker-compose -p`. I'd love to simplify this though, so if you have any suggestions feel free to pass them along.

The other option would be to convert this to a node based action, and then a simple node script "should" do the trick

Reply:
I think the Docker action is the right way to do this, but I don't think you need to go 2 levels deep.  The client API should "just work" if it is pointed at the right instance.  Have you tried setting the host to localhost and removing all of this 2 levels deep stuff?

Reply:
Offline discussion; turns out making this a `composite` action made it much simpler. No need for any of the docker-compose stuff